### ISSUE TITLE
Private Query Collection

### ISSUE THREAD

**Issue by fearless0307** [groveco/django-sql-explorer#472]
Each user can have a personal collection of private queries, which can be only edited or viewed by that user only.
@marksweb What is your opinion on this?

**Comment by marksweb** [comment]
Yeah that sounds like a good feature. Similar to #119 which suggests being able to mark a query as a favourite.

**Comment by fearless0307** [comment]
I will try to develop both features. Thanks.
